### PR TITLE
Prepare for 1.0.0-alpha release

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
$subject

### Issues

task

### Automation tests
 - Unit tests added: N/A
 - Integration tests added: N/A

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [-] Validated respective github issues
- [-] Assigned milestone to the github issue(s)

### CODE REVIEW

**Review by pubudu538 — APPROVED** [review]

File: release-note.html [hunk 1]
```diff
@@ -23,60 +23,43 @@
...
-            start guide</a></p>
+        <h1>What's new in 1.0.0-Alpha</h1>
+        <p>This is the alpha release of the Choreo Connect 1.0.0.</p>
         <h2>Features</h2>
         <ul>
-            <li> Choreo connect <a
-                    href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F8+is%3Aclosed+label%3A%22Type%2FNew+Feature%22">features</a></li>
+            <li>Deploy APIs from the file system</li>
+            <li>Support circuit breakers</li>
+            <li>Support retry for upstream endpoints</li>
```
Comment (SuKSW) [inline release-note.html:37]:
```suggestion
            <li>Support retry and timeout for upstream endpoints</li>
```

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211115.4

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211115.12

**Comment by choreo-cicd** [comment]
[failed] stage-deployment : 20211117.2

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment : 20211117.3

**Comment by choreo-cicd** [comment]
[succeeded] prod-deployment : 20211119.1

**Pull request merged**